packages/customize-widgets/src/components/inserter/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef, createPortal } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { __experimentalLibrary as Library } from '@wordpress/block-editor';
+import { Button } from '@wordpress/components';
+import { useInstanceId } from '@wordpress/compose';
+
+/**
+ * External dependencies
+ */
+import { Dialog, DialogBackdrop } from 'reakit/Dialog';
+
+function useTargetDOM() {
+	const target = useRef();
+	if ( ! target.current ) {
+		target.current = document.getElementById( 'widgets-left' );
```

Comment:
Rendering the inserter into a div that is in outside of `#wp-full-overlay-sidebar-content` makes sense to me.

I'm not sure that `#widgets-right` is the right place though as it seems that this ID only exists for backwards compatibility with plugins that look for widget templates inside a container called `#widgets-right`. Here we are adding things that are not widget templates.

The Customizer already has a slide-out drawer (it appears when you press the cog icon next to _Publish_) which exists in `#customize-outer-theme-controls`. Could we use this as the parent?